dot/core/service.go

```diff
@@ -79,15 +79,13 @@ type Config struct {
 	TransactionQueue TransactionQueue
 	Keystore         *keystore.Keystore
 	Runtime          *runtime.Runtime
-	MsgRec           <-chan network.Message
-	MsgSend          chan<- network.Message
 	NewBlocks        chan types.Block // only used for testing purposes
 	IsBabeAuthority  bool
 }
 
 // NewService returns a new core service that connects the runtime, BABE
 // session, and network service.
-func NewService(cfg *Config) (*Service, error) {
+func NewService(cfg *Config, msgRec <-chan network.Message, msgSend chan<- network.Message) (*Service, error) {
```

Comment:
why is this being changed? the reason we have a config is so we don't need to change the NewService function signature

Reply:
I was not sure whether to include the channels in the network service configuration or remove them from the core service configuration but they should be consistent. I chose to include the channels as params because they are created at the dot node level, same level as services, and they are channels between services.

I would not be strongly opposed to reverting and adding the channels to network config as an alternative but they should be consistent.

Reply:
okay, it would be preferred to add the network channels to their config as well

Reply:
see https://github.com/ChainSafe/gossamer/pull/699#discussion_r393318315

Reply:
`NewService` is not the same function as `createCoreService`, `NewService` should only take a Config

Reply:
I misread this the first time and will fix.

Reply:
no worries! sounds good